src/server-extension/resolvers/index.ts

```diff
@@ -4,8 +4,10 @@ import { OfferStatsResolver } from './offerStatsResolver';
 import { EventResolver } from './event';
 import { SeriesResolver } from './series';
 import { SpotlightResolver } from './spotlight';
+// import { CollectionResolver } from './collectionResolver';
```

Comment:
Should remove unused imports

Reply:
ohh good catch :+1: forgot to save file after removing, was doing some testing